platform/use-platform/virtual-clusters/key-features/snapshots.mdx

```diff
@@ -13,13 +13,16 @@ vCluster Platform allows you to configure taking snapshots of the vCluster at sp
 This allows administrators to capture and store the vCluster state in scheduled intervals to help protect 
 against infrastructure failures, data corruption, and configuration errors. By maintaining consistent 
 recovery points, administrators can quickly restore the vCluster to a known good state without relying on manual backup processes. 
-For more details on how snapshots work, refer to the documentation in the [Snapshot and Restore](https://www.vcluster.com/docs/vcluster/manage/backup-restore/) section.
+For more details on how snapshots work, refer to the documentation in the [Snapshot and Restore](/vcluster/manage/backup-restore/) section.
 
 In the `vcluster.yaml`, it is configured under `external.platform.autoSnapshot`. Using the UI, you 
 can configure the management of snapshots in the config options of a virtual cluster under **Snapshots**. Though 
 snapshot configuration is configured on the virtual cluster itself, the capability and logic of scheduling snapshots
 is in vCluster Platform. 
 
+:::note
+Auto Snapshot is supported from platform version 4.4.0 onward and is currently in Beta.
```

Comment:
Auto Snapshot itself is GA, only the volumes part is beta. Since there is a beta callout below, we can just talk about vanilla feature here.